src/core/gui/widgets/SpinPageAdapter.h

```diff
@@ -51,7 +53,7 @@ class SpinPageAdapter {
     gulong pageNrSpinChangedHandlerId = 0;
     size_t page = 0;
 
-    guint lastTimeoutId = 0;
+    xoj::util::GSourceURef timout;
```

Comment:
I guess you intended to call this variable `timeout` here and elsewhere.
```suggestion
    xoj::util::GSourceURef timeout;
```